src/pages/About.jsx

```diff
@@ -9,76 +9,46 @@ import {
+import { useTranslation, Trans } from 'react-i18next';
 import backgroundImg from '../img/img6.svg';
 import LookingForScholarshipsBanner from '../components/LookingForScholarshipsBanner';
 
-const team = [
-  {
-    name: 'Josue Rios',
-    img: 'https://avatars.githubusercontent.com/u/8023233?v=4',
-    description: (
-      <>
-        I am a DACA recipient currently working as a software engineer. I came
-        up with this project idea after participating in the{' '}
-        <MuiLink href="http://beyondhb1079.org/">Beyond HB 1079</MuiLink> team
-        and noticing how painful it is to find scholarships and compile a list
-        for undocumented students. My hope is to make this the go-to resource
-        for scholarships for undocumented students.
-      </>
-    ),
-  },
-  {
-    name: 'Edwin Lopez',
-    img: 'https://avatars.githubusercontent.com/u/40483569?v=4',
-    description: (
-      <>
-        I am a software engineer from the University of California Irvine. I
-        enjoy learning and using my skills to make software that makes an impact
-        in communites. Being a DACA recipient has led me to have an emotional
-        connection with {BRAND_NAME}.
-      </>
-    ),
-  },
-  {
-    name: 'Gonzalo Lara',
-    img: 'https://avatars.githubusercontent.com/u/64123425?v=4',
-    description: (
-      <>
-        My passion for computers has lead me to pursue a career in Software
-        Engineering. I am currently a UWB CSS student completing my final year
-        in the Computer Science and Software Engineering program. I am 24 years
-        old, bilingual and currently a part of the DACA program. I intend to
-        contribute as much as possible to building a better tomorrow.
-      </>
-    ),
-  },
-  {
-    name: 'Sergio Mejia',
-    img: 'https://avatars.githubusercontent.com/u/15769145?v=4',
-    description: (
-      <>
-        I’m a UX Designer that is interested in using technology as a means to
-        solve problems. My personal experiences have inspired my focus on
-        empathy and universal design. I find joy in the ability to explore ideas
-        with a central aim to create solutions that will help people solve
-        meaningful problems.
-      </>
-    ),
-  },
-  {
-    name: 'Job Hernandez',
-    img: 'https://avatars.githubusercontent.com/u/69167740?v=4',
-    description: (
-      <>
-        I am driven to solve problems with software; software has tremendous
-        power to advance society. So, I would like to contribute and have an
-        impact.
-      </>
-    ),
-  },
-];
-
 function About() {
+  const { t } = useTranslation();
+
+  const team = [
+    {
+      name: 'Josue Rios',
+      img: 'https://avatars.githubusercontent.com/u/8023233?v=4',
+      description: (
+        <Trans i18nKey="about.team.josue">
+          start
+          <MuiLink href="http://beyondhb1079.org/">Beyond HB 1079</MuiLink> end
+        </Trans>
```

Comment:
[Trans](https://react.i18next.com/latest/trans-component) component breaks up the text into the following parts
1 -> start
2 -> `<MuiLink> </MuiLink>`
3 -> end

I could have left the oiginal text instead of `start/end` but I thought those saved some space.